doc/install.txt

```diff
@@ -412,7 +412,7 @@ that we use.
 3) Install the ATLAS library. ATLAS is an open source optimized version of
 BLAS. You can install a precompiled version on most OSes, but if you're willing
 to invest the time, you can compile it to have a faster version (we have seen
-speed-ups of up to 3x, especialy on more recent computers, against the
+speed-ups of up to 3×, especially on more recent computers, against the
```

Comment:
In this case I don't think the slight visual improvement is worth it. There is always the risk that browsers (especially on windows) will not have a proper Unicode font and will display a box instead of an x.

Also what kind of problems do you encounter with accents in NEWS.txt?